public/styles/styles.css

```diff
@@ -146,7 +146,99 @@ body {
     text-align: center;
     text-shadow: 2px 2px 30px var(--red);
 }
+/* Footer begins here */
+.footerbar-container{
+    background-color: var(--light-grey);
+    height: auto;
+    width: auto;
+}
+
+.footer-socialmedia{
+    margin:  -120px 0px 0px 1035px;  
+    padding: 2px; 
+}
+
+.footer-socialmedia > a{
+    text-decoration-line: none;
+    font-weight: bold;
+}
+
+.symbol{
```

Comment:
@ZM202 There might be another symbol class name in the stylesheet. So to prevent any issues in the feature, chain the class name so that this symbol is associated with the social media section of the footer. Can name it `.footer-socialmedia .symbol`